andersoncd/penalties.py

```diff
@@ -146,3 +146,41 @@ def subdiff_distance(self, w, grad, ws):
 
     def is_penalized(self, n_features):
         return self.weights != 0
+
+
+spec_MCP = [
+    ('alpha', float64),
+    ('gamma', float64),
+]
+
+
+@jitclass(spec_L1)
+class MCP(Penalty):
+    def __init__(self, alpha, gamma):
+        self.alpha = alpha
+        self.gamma = gamma
+
+    def value(self, w):
+        return MC_penalty(w, self.alpha, self.gamma)
```

Comment:
you can implement it right here instead of importing it (right now I don't see what other part of the code would need to access it, justifying it being in a separate file)